css/custom.css

```diff
@@ -211,14 +260,29 @@ h1.title{
   width:276px;
 }
 
-.form-item-langcode{
+.form-item-langcode,
+.node-digital-object-collection-form .form-type-language-select{
```

Comment:
Hi @natehill, this language hiding needs to go. Reason is Archipelago is a multi lingual repository, this breaks any option to have that and if for any reason the logic requires language this will just end in a form that can't be validated but can also not be corrected.. If you don't want the language in the demo theme you can remove the language field from the default form display in "manage displays"

Reply:
ok i made the change and will open a new pull request but i cannot for the life of me find where to 'remove the language field from the default form display in "manage displays"', friggin drupal UI maze

Reply:
Ok, will follow up with you via email on that. Thanks for the update